titus-api/src/test/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobAssertionsTest.java

```diff
@@ -29,13 +31,40 @@
 
     private final JobConfiguration configuration = mock(JobConfiguration.class);
 
+    private final JobAssertions jobAssertions = new JobAssertions(configuration, id -> ResourceDimension.empty());
+
+    @Test
+    public void testEmptyMapOfEnvironmentVariableNames() {
+        Map<String, String> violations = jobAssertions.validateEnvironmentVariableNames(Collections.emptyMap());
+        assertThat(violations).isEmpty();
+    }
+
+    @Test
+    public void testGoodEnvironmentVariableNames() {
+        Map<String, String> violations = jobAssertions.validateEnvironmentVariableNames(ImmutableMap.of(
+                "THIS_IS_GOOD_123", "good value",
+                "_AND_THIS", "good value"
+        ));
+        assertThat(violations).isEmpty();
+    }
+
+    @Test
+    public void testBadEnvironmentVariableNames() {
+        Map<String, String> violations = jobAssertions.validateEnvironmentVariableNames(ImmutableMap.of(
```

Comment:
add `_THIS_IS_2_GOOD` and `1_THIS_IS_NOT_GOOD` for completeness?

Reply:
Added